### DISCUSSION TITLE
Dietpi-Software | Speedtest Tracker

### DISCUSSION THREAD

**Discussion by caio1007** [MichaIng/DietPi#5905] [Software Request]
Hello, I would like to make a request for the future implementation of this software, I can use it via docker, but I would like a native installation in the future by Dietpi-Software

This program runs a speedtest check every hour and graphs the results. The back-end is written in [Laravel](https://laravel.com/) and the front-end uses [React](https://reactjs.org/). It uses the [Ookla's speedtest cli](https://www.speedtest.net/apps/cli) package to get the data and uses [Chart.js](https://www.chartjs.org/) to plot the results.

A demo can be found [here](https://speedtest.henrywhitaker.com/)

[Link Software](https://github.com/henrywhitaker3/Speedtest-Tracker)

**Comment by mews-se** [comment]
I agree. However I would recommend this fork that is better maintained!

https://github.com/alexjustesen/speedtest-tracker

**Reply by Joulinar** [reply]
> I agree. However I would recommend this fork that is better maintained!
>
> https://github.com/alexjustesen/speedtest-tracker

This one seems to be based on Docker and should be easy to install yourself. From DietPi side we don't go to add Docker container to our software catalogue actually.

**Comment by mews-se** [comment]
There is the complete source code available as well. And as he states in his README, henrywhitaker has abandoned the older one

**Comment by caio1007** [comment]
it would be a great implementation in dietpi-software that version you mentioned. I use raspberry pi 4 and docker consumes more resources than installed directly on the system.
